README.md

```diff
@@ -181,37 +184,208 @@ Other available options:
+
+#### Openstack example
+
+```
+network create --OPENSTACK --name openstacknetwork --description openstack-network --publicNetID "324234-32423423-23423423-23423432" --subnet 0.0.0.0
+```
+
+#### Azure example
+
+```
+network create --AZURE --name azurenetwork --description azure-network --addressPrefix 192.168.123.123 --subnet 0.0.0.0
+```
+
+Other available options:
+
+- --publicInAccount "flag": flags if the template is public in the account
+
+#### Viewing created network
+
+You can check whether the network was created successfully by using the `network list` command. Check the network and select it if you are happy with it:
+
+```
+network show --id <ID of the network>
+```
+or
+```
+network show --name <Name of the network>
+```
+
+### Configure an instance groups
+
+You have to configure the instancegroups before the provisioning. An instancegroup is defining a group of your nodes with a specified templates.
+
+```
+instancegroup configure --instanceGroup slave_1 --nodecount 1 --templateId 51
+```
+
+Other available options:
+
+- --templateName "string": Name of the template
+
+### Configure a host groups
+
+You have to configure the host groups before the provisioning(by default the cloudbreak shell doing it automatically). A host group is defining a host group in you ambari cluster.
+
+```
+hostgroup configure --hostgroup slave_1 --recipeIds 1,2,3 
+```
+
+### Configure a recipe
+
+You can create recipes for automate your setups before ambari install or after ambari install.
 
 ```
-stack create --name “myStackName" --nodeCount 10
+recipe --url "https://raw.githubusercontent.com/sequenceiq/consul-plugins-titan/master/titan-recipe.json"
 ```
+
+Other available options:
+
+- --file "string": the file which contains the recipe
+- --publicInAccount "flag": flags if the template is public in the account
+
 ### Select a blueprint
 
 We ship default Hadoop cluster blueprints with Cloudbreak. You can use these blueprints or add yours. To see the available blueprints and use one of them please use:
 
 ```
 blueprint list
+```
+then select one of the created blueprints
+
+```
+blueprint select --id <ID of the blueprint>
+```
+or
+```
+blueprint select --name <Name of the blueprint>
+```
+
+### Create a stack
+
+Stacks are template `instances` - a running cloud infrastructure created based on a template. Use the following command to create a stack to be used with your Hadoop cluster:
+
+```
+stack create --name myStackName --region EU_WEST_1
+```
+
+Other available options:
+
+- --image "string": the specific image which is used under the provisioning
+- --onFailureAction "string": the available values are DO_NOTHING or ROLLBACK
+- --adjustmentType "string": the available values are EXACT or PERCENTAGE
+- --threshold "long": the number for the adjustmentType
+- --diskPerStorage "integer": the number of disks in one storage (only for AZURE)
+- --dedicatedInstances "flags": the flag if you want to use dedicated instances (only for AWS)
```

Comment:
instead of: the flag if you want to use dedicated instances (only for AWS)
use: indicates using dedicated instances (AWS only